src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotificationDefendantOfHearingHandler.java

```diff
@@ -90,17 +73,59 @@ public List<CaseEvent> handledEvents() {
 
     @Override
     public Map<String, String> addProperties(final CaseData caseData) {
-        String hourMinute = caseData.getHearingTimeHourMinute();
-        int hours = Integer.parseInt(hourMinute.substring(0, 2));
-        int minutes = Integer.parseInt(hourMinute.substring(2, 4));
-        LocalTime time = LocalTime.of(hours, minutes, 0);
+        String legacyCaseRef = caseData.getLegacyCaseReference();
+        String hearingDate = NotificationUtils.getFormattedHearingDate(caseData);
+        String hearingTime = NotificationUtils.getFormattedHearingTime(caseData);
         return new HashMap<>(Map.of(
             CLAIM_REFERENCE_NUMBER,
-            caseData.getLegacyCaseReference(),
+            legacyCaseRef,
             HEARING_DATE,
-            caseData.getHearingDate().format(DateTimeFormatter.ofPattern("dd-MM-yyyy")),
+            hearingDate,
             HEARING_TIME,
-            time.format(DateTimeFormatter.ofPattern("hh:mma")).replace("AM", "am").replace("PM", "pm")
+            hearingTime
         ));
     }
+
+    private boolean isRespondentLip(CaseData caseData) {
+        return YesOrNo.NO.equals(caseData.getRespondent1Represented());
+    }
+
+    private String getRespondentRecipient(CaseData caseData, boolean isDefendant1, boolean isRespondentLip) {
```

Comment:
in sonar is saying 
![image](https://github.com/hmcts/civil-service/assets/93268657/761e1910-3cae-4887-969e-c8b46dcee025)

Reply:
as well is difficult to read this part.
My suggestion 
        if  (isDefendant1) {
            return isRespondentLip ? caseData.getRespondent1().getPartyEmail()
                : caseData.getRespondentSolicitor1EmailAddress();
        } else {
            if (isRespondentLip){
                if (nonNull(caseData.getRespondent2().getPartyEmail())){
                    return caseData.getRespondent2().getPartyEmail();
                }
                return caseData.getRespondent1().getPartyEmail();
            } else {
                if (nonNull(caseData.getRespondentSolicitor2EmailAddress())){
                    return caseData.getRespondentSolicitor2EmailAddress();
                }
                return caseData.getRespondentSolicitor1EmailAddress()
            }
        }
        return null;

Reply:
Yes! I'll refactor it.